analyses/mutation-signatures/util/mut_sig_functions.R

```diff
@@ -0,0 +1,209 @@
+# Functions for making mutational signature plots and calculations
+#
+# C. Savonen for ALSF - CCDL
+# 2019
+
+# Load this library
+library(deconstructSigs)
+
+################################################################################
+sample_mut_sig_plot <- function(which_sig_list, label = "none", output_dir = getwd(),
+                                sample_ids) {
```

Comment:
`sample_ids` is required, right? If so, keep it in the first set of arguments, and put the ones with default values later.